tests/ludwig/schema/test_model_config.py

```diff
@@ -784,3 +788,42 @@ def test_encoder_decoder_values_as_str():
 
     assert isinstance(config_obj.input_features[0].encoder, BERTConfig)
     assert isinstance(config_obj.output_features[0].decoder, ClassifierConfig)
+
+
+@pytest.mark.parametrize(
+    "base_model_config,model_name",
+    [
+        ("bloomz-3b", "bigscience/bloomz-3b"),
+        ("llama-7b", "huggyllama/llama-7b"),
+    ],
+)
+def test_llm_base_model_config(base_model_config, model_name):
+    config = {
+        MODEL_TYPE: MODEL_LLM,
+        BASE_MODEL: base_model_config,
+        INPUT_FEATURES: [{NAME: "text_input", TYPE: "text"}],
+        OUTPUT_FEATURES: [{NAME: "text_output", TYPE: "text"}],
+    }
+
+    config_obj = ModelConfig.from_dict(config)
+
+    assert config_obj.base_model == model_name
```

Comment:
Can we add a similar test for a fully-qualified name and make sure the custom `AutoConfig` check works?

Reply:
Added one more test config with a fully-qualified name that exists. The test below this one checks that the logic raises an error if the path is invalid.